docs/LoaderLayerInterface.md

```diff
@@ -0,0 +1,1732 @@
+information provided by these default paths.
+
+For security reasons, `VK_LAYER_PATH` is ignored if running with elevated
+privileges.
+See [Exception for Elevated Privileges](#exception-for-elevated-privileges)
+for more info.
+
+**NOTE** While the order of folders searched for manifest files is well
+defined, the order contents are read by the loader in each directory is
+[random due to the behavior of readdir](https://www.ibm.com/support/pages/order-directory-contents-returned-calls-readdir).
+
+See
+[Forcing Layer Source Folders](LoaderApplicationInterface.md#forcing-layer-source-folders)
+in the [LoaderApplicationInterface.md document](LoaderApplicationInterface.md)
+for more information on this.
+
+
+#### Example Linux Explicit Layer Search Path
+
+```
+  /home/me/.config/vulkan/explicit_layer.d
+  /etc/xdg/vulkan/explicit_layer.d
+  /usr/local/etc/vulkan/explicit_layer.d
+  /etc/vulkan/explicit_layer.d
+  /home/me/.local/share/vulkan/explicit_layer.d
+  /usr/local/share/vulkan/explicit_layer.d
+  /usr/share/vulkan/explicit_layer.d
+```
+
+### Fuchsia Layer Discovery
+
+On Fuchsia, the Vulkan loader will scan for manifest files using environment
+variables or corresponding fallback values if the corresponding environment
+variable is not defined in the same way as [Linux](#linux-layer-discovery).
+The **only** difference is that Fuchsia does not allow fallback values for
+*$XDG_DATA_DIRS* or *$XDG_HOME_DIRS*.
+
+
+### macOS Layer Discovery
+
+On macOS, the Vulkan loader will scan for manifest files using the application
+resource folder as well as environment variables or corresponding fallback
+values if the corresponding environment variable is not defined.
+The order is similar to the search path on Linux with the exception that
+the application's bundle resources are searched first:
+`(bundle)/Contents/Resources/`.
+
+#### Example macOS Implicit Layer Search Path
+
+```
+  <bundle>/Contents/Resources/vulkan/implicit_layer.d
+  /Users/Me/.config/vulkan/implicit_layer.d
+  /etc/xdg/vulkan/implicit_layer.d
+  /usr/local/etc/vulkan/implicit_layer.d
+  /etc/vulkan/implicit_layer.d
+  /Users/Me/.local/share/vulkan/implicit_layer.d
+  /usr/local/share/vulkan/implicit_layer.d
+  /usr/share/vulkan/implicit_layer.d
+```
+
+### Exception for Elevated Privileges
+
+There is an exception to when `VK_LAYER_PATH` is available for use.
+For security reasons, `VK_LAYER_PATH` is ignored if running the Vulkan
+application with elevated privileges.
+Because of this, `VK_LAYER_PATH` can only be used for applications that do not
+use elevated privileges.
+
+For more information see
+[Elevated Privilege Caveats](LoaderInterfaceArchitecture.md#elevated-privilege-caveats)
+in the top-level
+[LoaderInterfaceArchitecture.md][LoaderInterfaceArchitecture.md] document.
+
+## Layer Version Negotiation
+
+Now that a layer has been discovered, an application can choose to load it (or
+it is loaded by default if it is an Implicit layer).
```

Comment:
I like this instead: 
`Now that a layer has been discovered, an application can choose to load it, or in the case of implicit layers, it can be loaded by default.`